netprofile_core/netprofile_core/models.py

```diff
@@ -7144,6 +7145,25 @@ def __str__(self):
                           str(self.type))
 
 
+def user_setting(user, name):
+
```

Comment:
Make this function consistent with global_setting(). In particular, maintain same execution order.